src/defaultsettings.cpp

```diff
@@ -206,6 +206,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("invert_mouse", "false");
 	settings->setDefault("mouse_sensitivity", "0.2");
 	settings->setDefault("repeat_rightclick_time", "0.25");
+	settings->setDefault("safe_dig_and_place", "false");
```

Comment:
Maybe ignore the inconsistency and set this one to `true` by default for Android players?